contracts/MerkleDistributor.sol

```diff
@@ -44,21 +52,16 @@ contract MerkleDistributor is IMerkleDistributor {
 
          // Mark it claimed and send the token.
         _setClaimed(index);
-        // [P] Start (unix): 1607990400 | Tuesday, December 15th, 2020 @ 12:00AM GMT
-        uint256 startTime = 1607495500;
-        // [P] End (unix): 1616630400 | Thursday, March 25th, 2021 @ 12:00AM GMT
-        uint256 endTime = 1616630400;       
-        uint256 nowTime = block.timestamp;
-        uint256 duraTime = nowTime - startTime;
-        require(nowTime >= startTime, 'MerkleDistributor: Too soon');
-        require(nowTime <= endTime, 'MerkleDistributor: Too late');
+        uint256 duraTime = now.sub(startTime);
+        require(now >= startTime, 'MerkleDistributor: Too soon');
+        require(now <= endTime, 'MerkleDistributor: Too late');
         // create a ceiling for the maximum amount of duraTime
-        uint256 duraDays = duraTime / 86400 >= 90 ? 90 : duraTime / 86400; // divided by the number of seconds per day
-        require(duraDays <= 100, 'MerkleDistributor: Too late'); // double check days
-        uint256 availAmount = amount * (10 + duraDays) / 100;// 10% + 1% daily
+        uint256 duraDays = duraTime.div(secondsInaDay); // divided by the number of seconds per day
+        require(duraDays <= 90, 'MerkleDistributor: Too late'); // double check days
```

Comment:
Claimants have +10 days after the end of airdrop (day 91-100) to claim their reward, the duraDays checks/buffers need to support that

Reply:
Fixed

Reply:
thank you so much for the assistance, makes a lot more sense now.